datasetComparison/src/test/scala/za/co/absa/hermes/datasetComparison/ComparisonResultSuite.scala

```diff
@@ -21,12 +22,13 @@ class ComparisonResultSuite extends FunSuite {
   }
 
   test("testGetMetadata") {
-    val CR2 = ComparisonResult(0, 0, 0, 0, List.empty[Column], None, 0, "--alfa beta")
+    val CR2 = ComparisonResult(0, 0, 0, 0, 0, List.empty[Column], None, 0, "--alfa beta")
     val result = Map(
       "passed" -> "true",
-      "numberOfDuplicates" -> "0",
+      "refDuplicateCount" -> "0",
```

Comment:
How about a test case with some duplicates?

Reply:
In the case of ComparisonResult class, nothing changes. And in case of DatasetComparisonJob, this will be handled on the integration level because you need to change config.